### PR TITLE
Resolve out-of-space issue in azure pipelines CI

### PR DESCRIPTION
**Pull request by kennyweiss**
# Summary

- Attempt to resolve 'out-of-space' issue in azure pipelines CI (#214)
- Sets 'clean' step to 'true' in azure-pipelines

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -67,7 +67,7 @@ pool:
 
 steps:
 - checkout: self
-  clean: boolean
+  clean: true
```
Comment (white238) [inline azure-pipelines.yml:70]: I did this in my PR and it didnt solve the problem.

Comment (kennyweiss) [reply azure-pipelines.yml:70]: Thanks @white238 -- I didn't see that you already tried.

The next step might be to wrap our ``steps`` task under a ``jobs`` task, and add a ``clean:output`` to the ``job``.

E.g.:

```
jobs:
- job: MyJob
  displayName: My First Job
  continueOnError: true
  workspace:
    clean: outputs                         # <-- This
  steps:
  - script: echo My first job
```
https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=example%2Cparameter-schema#job

Comment (white238) [reply azure-pipelines.yml:70]: I also added a make clean in my repo that dropped the build directory from 220megs to 9.. and that made no difference.

**Comment by kennyweiss** [comment]
@white238 resolved this in #212 by disabling the ``gcc-6`` build. 
We might have found (part of) the problem in #216.

**Pull request closed without merging**